lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_check_devices.rb

```diff
@@ -0,0 +1,66 @@
+module Fastlane
```

Comment:
This looks like a solid action, but I can't see it being called from anywhere 🙈 

Am I just missing something? 🙃

Reply:
🙂 This action is used in this [pull request](https://github.com/wordpress-mobile/WordPress-Android/pull/10841).